openslides/users/static/js/users/pdf.js

```diff
@@ -213,11 +213,11 @@ angular.module('OpenSlidesApp.users.pdf', ['OpenSlidesApp.core.pdf'])
             var createWelcomeText = function() {
                 return [
                     {
-                        text:   getConfigValue('users_pdf_welcometitle'),
+                        text:   gettextCatalog.getString(getConfigValue('users_pdf_welcometitle')),
```

Comment:
if you want to rename this function in pdf please suggest a new name. otherwise please approve